.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: '1.22.2'
+          go-version: '>=1.23.0'
```

Comment:
```suggestion
          go-version: '1.23.0'
```

Reply:
Bump Go version to v1.23 to match the k8s/k8s repo.
https://github.com/kubernetes/kubernetes/blob/master/.go-version